test/pkgcheck.sh

```diff
@@ -125,7 +125,7 @@ cd btmp1
     ;;
   esac
   ../configure $CONFIGURE_ARGS
-  make
+  make -j2
```

Comment:
Technically `make -j` (without any number) would also work here and leverage computers with higher core counts as well.

Reply:
Fair enough. It also does not really matter for the speed as the configuration steps are single-threaded and are the major bottlenecks.